### ISSUE TITLE
Record party in playlist

### ISSUE THREAD

**Issue by NeoLegends** [Festify/app#126]
From Twitter user @hokeypunk33

> @GetFestify Any chance we'll get the option to save the parties tracks to a playlist?

**Comment by OliverCullimore** [comment]
How would be best to implement this?
- As a history (everytime a track is played it is added to the playlist)
- As a live copy (everytime a song is added to the queue it is added to the playlist)
Would this just include voted for tracks or the fallback too. 
If the live copy - Would this need to take into account the order of the tracks from the voting?

**Comment by leolabs** [comment]
I'd implement it as a history of played songs because the live queue of songs that may be played in the future can get quite big and filled with songs that aren't relevant.

**Comment by OliverCullimore** [comment]
I agree @leolabs that seems the sensible option. Does anyone have any detailed instructions on how to setup a dev environment for this? The dependencies in the readme state you need to setup a dedicated and paid firebase project which I can't find an option for on the firebase site? Are the instructions out of date at all? Also will it cost much to run the development environment? Additionally I couldn't see the config files to enter the details into when I cloned the project, do I need to create these myself rather than just edit them?

**Comment by leolabs** [comment]
Hey @OliverCullimore,

to create a paid Firebase project, just create a normal Firebase project first and then upgrade it to "Pay-as-you-go". You won't have to pay much, a few cents at most. We currently have 30k active monthly users and pay about 3€ per month.

The config files have to be created manually. Please refer to the readme which describes what information is needed.

If you have further questions, please don't hesitate to ask :)

**Comment by mrclschstr** [comment]
Adding played songs to a separate Spotify playlist would be a very nice feature 👍
